named_entity_recognition_bert/named_entity_recognition_huggingface_bert_pipeline.ipynb

```diff
@@ -0,0 +1,1124 @@
+    "\n",
+    "data_links_dict = {'train': [],\n",
+    "                   'unlabeled': []}\n",
+    "\n",
+    "BUCKET_FOLDER_PATH = '/path/to/data/'\n",
+    "\n",
+    "start_key = ''\n",
+    "\n",
+    "for subset_name in subset_names:\n",
+    "    print(\"Processing\", subset_name)\n",
+    "    while True:\n",
+    "        response = s3_client.list_objects_v2(Bucket=bucket_name,\n",
+    "                                             Prefix=f'{BUCKET_FOLDER_PATH}/{subset_name}/',\n",
+    "                                             StartAfter=start_key)\n",
+    "        objects = response['Contents']\n",
+    "        for obj in objects:\n",
+    "            data_links_dict[subset_name].append(obj['Key'])\n",
+    "        print(f\"\\t{len(data_links_dict[subset_name])} files in {subset_name}\")\n",
+    "        start_key = objects[-1]['Key']\n",
+    "        if len(objects) < 1000:\n",
+    "            start_key = ''\n",
+    "            break"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c191650a",
+   "metadata": {},
+   "source": [
+    "### 2.2. Download files\n",
+    "<a id='download_files'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "892942fe-9b07-4e01-b0d1-b8c4b60920fb",
+   "metadata": {},
+   "source": [
+    "Now we will use these links to download all the files from S3 bucket."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "216dfcbf",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "for subset_name in subset_names:\n",
+    "    print(f\"Loading {subset_name} docs\")\n",
+    "    save_dir = f'./{subset_name}_sa_docs'\n",
+    "    if not os.path.exists(save_dir):\n",
+    "        os.mkdir(save_dir)\n",
+    "    for file_key in tqdm(data_links_dict[subset_name]):\n",
+    "        if not '.txt' in file_key:\n",
+    "            continue\n",
+    "        filename = os.path.basename(file_key)\n",
+    "        s3_client.download_file(Bucket=bucket_name, \n",
+    "                                Key=file_key,\n",
+    "                                Filename=os.path.join(save_dir, filename))\n",
+    "        "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6e36049c",
+   "metadata": {},
+   "source": [
+    "### 2.3 Download labels from SuperAnnotate\n",
+    "<a id='download_labels_from_sa'></a>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4c569da6-7eb9-4a62-afef-8764aa2fce1a",
+   "metadata": {},
+   "source": [
+    "Now we can download labels from SuperAnnotate for the train texts that were annotated manually. The annotations will be downloaded in [SuperAnnotate format](https://doc.superannotate.com/docs/sdk-export-annotations)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bff0e565",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "token = \"e7514d15a4e0a2bc3824e864bada3aa12183c2aff6ce8910cd14f719946cc6ba797b17ddb4d0b651t=19366\"\n",
```

Comment:
PUT_YOUR_TOKEN_HERE and comments with link to documentation with getting a token part